app/controllers/users_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+
+  def index
+    @users = User.all
+
+    respond_to do |format|
+      format.html
+      format.json { render json: @users }
+    end
+  end
+
+  def show
+    @user = User.find(params[:id])
+
+    respond_to do |format|
+      format.html
+      format.json { render json: @user }
+    end
+  end
+
+  def update
+    @user = User.find(params[:id])
+    @user.update(user_params)
```

Comment:
You have to do here something similar to what you did in index and show. And again, it seems to me that you are not testing if this is running or not, there are no views for these actions.